src/app/components/ConfigProvider.tsx

```diff
@@ -19,8 +19,8 @@
 
 'use client';
 
+import { AppConfig } from '@/app/api/config/config';
```

Comment:
Following up on the TBD absolute paths topic: Another example of what could be aliased as a shortcut: `@/api`, as it shouldn't matter that it's in `app` (unless it does? I doubt it)